### PR TITLE
Bugfix/crop production 182 negative production vals

### PR DESCRIPTION
**Pull request by richpsharp**
This fixes a critical error in crop production regression that was just straight up calculating the wrong results. Also fixed a few typos, updated history and updated test data. Also, here's the PR for the updated regression data: https://bitbucket.org/natcap/invest-test-data/pull-requests/12/replacing-regression-data-for-crop/diff

### CODE REVIEW

File: src/natcap/invest/crop_production_regression.py [hunk 1]
```diff
@@ -706,7 +706,7 @@ def tabulate_regression_results(
         landcover_raster_path (string): path to landcover raster
         landcover_nodata (float): landcover raster nodata value
         output_dir (string): the file path to the output workspace.
-        file_suffix (string): string to appened to any output filenames.
+        file_suffix (string): string to appended to any output filenames.
```
Comment (dcdenu4) [inline src/natcap/invest/crop_production_regression.py:709]: Typo: string to append to any output filenames

File: src/natcap/invest/crop_production_regression.py [hunk 2]
```diff
@@ -612,18 +612,18 @@ def _x_yield_op(
...
     valid_mask = (
         (b_x != _NODATA_YIELD) & (c_x != _NODATA_YIELD) &
         (lulc_array == crop_lucode))
-    result[valid_mask] = y_max[valid_mask] * (
+    result[valid_mask] = pixel_area_ha * y_max[valid_mask] * (
         1 - b_x[valid_mask] * numpy.exp(
-            -c_x[valid_mask] * fert_rate) *
-        pixel_area_ha)
+            -c_x[valid_mask] * fert_rate))
+
```
Comment (dcdenu4) [inline src/natcap/invest/crop_production_regression.py:626]: Ah, yea, don't want that pixel to area conversion near that `1 - ` operation.

File: src/natcap/invest/crop_production_regression.py [hunk 3]
```diff
@@ -808,7 +808,7 @@ def aggregate_regression_results_to_polygons(
         nutrient_table (dict): a lookup of nutrient values by crop in the
             form of nutrient_table[<crop>][<nutrient>].
         output_dir (string): the file path to the output workspace.
-        file_suffix (string): string to appened to any output filenames.
+        file_suffix (string): string to appended to any output filenames.
```
Comment (dcdenu4) [inline src/natcap/invest/crop_production_regression.py:811]: typo again: `appended` -> `append`

**Comment by richpsharp** [comment]
Hi All, I thought I'd be tricky and push the branch right to the main repo for PR consideration. That only works once apparently and I am unable to push any updates to it. I'm going to close this PR so I can fork invest so I can make a branch with these updates and then PR it back in.

**Pull request closed without merging**